src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  font-family: Roboto, system-ui;
```

Comment:
The font-family declaration should include a generic font family as a fallback. For instance, 'Roboto, sans-serif'. This ensures text is displayed with a default font even if the primary font cannot be loaded.